examples/dotnet/FunctionExecutionCs/README.md

```diff
@@ -0,0 +1,27 @@
+﻿# PutGetRemove Example
```

Comment:
this readme still looks like it is a copy of another example...

Reply:
README updated.